modules/rostests/winetests/iphlpapi/iphlpapi.c

```diff
@@ -71,9 +71,34 @@ static DWORD (WINAPI *pNotifyUnicastIpAddressChange)(ADDRESS_FAMILY, PUNICAST_IP
                                                 PVOID, BOOLEAN, HANDLE *);
 static DWORD (WINAPI *pCancelMibChangeNotify2)(HANDLE);
 
+#if defined(__REACTOS__) && DLL_EXPORT_VERSION < 0x600
```

Comment:
```suggestion
#if defined(__REACTOS__) && (DLL_EXPORT_VERSION < _WIN32_WINNT_VISTA)
```
\+ same suggestion for all the other occurrences in this file.